hyperspy/io.py

```diff
@@ -512,10 +514,10 @@ def assign_signal_subclass(dtype, signal_dimension, signal_type="", lazy=False):
         Signal dtype
     signal_dimension : int
         Signal dimension
-    signal_type : str, default ""
-        Signal type. Optional. Will log a warning if it is unknown to HyperSpy.
-    lazy : bool, default False
-        If True, returns the matching LazySignal subclass.
+    signal_type : str, optional
+        Signal type. Default ''. Will log a warning if it is unknown to HyperSpy.
+    lazy : bool, optional
+        If True, returns the matching LazySignal subclass. Default is False.
 
     Returns
     -------
```

Comment:
Add `list of signal`?

Reply:
Sorry, I misread... I thought that this was still the docstring of the `load` function!